src/client/renderer/r_atlas.h

```diff
@@ -31,35 +31,37 @@ void R_CompileAtlas(r_atlas_t *atlas);
 #ifdef __R_LOCAL_H__
 
 typedef struct {
-	uint32_t width;
-	uint32_t height;
-	uint32_t x;
-	uint32_t y;
+	uint16_t width;
+	uint16_t height;
+	uint16_t x;
+	uint16_t y;
 
 	uint32_t right;
 	uint32_t down; // nodes to the right/below this one
 } r_packer_node_t;
```

Comment:
Wouldn't mind seeing these structs named `r_atlas_packer_t`, `r_atlas_packet_node_t`, etc..

Reply:
I was also wondering if maybe we should just move the atlas packer to libimage or something, and call them Img_Atlas*. We only ever use the packer for images.

Reply:
I guess the question is: would we ever build an atlas outside of the renderer? Would Quemap potentially bake these lightmap atlases for us? Probably not, because they depend on your lightmap block size cvar, right? So, assuming this is a renderer-only concern, it should stay in the renderer imo.

Reply:
Cgame uses it to compile an atlas, but that's about it.